src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * contains the task list
+ */
+public class TaskList {
+    private final ArrayList<Task> listOfTasks;
+
+    /**
+     * Constructor for creating TaskList object
+     *
+     * @param listOfTasks list of input tasks
+     */
+    public TaskList(ArrayList<Task> listOfTasks) {
+        this.listOfTasks = listOfTasks;
+    }
+
+    /**
+     * Alternative constructor for empty list
```

Comment:
Perhaps should put a fullstop after the sentence? Found this issue at several places as well